cps2pc

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python
+
+import argparse
+import cps
+from collections import OrderedDict
+import os
+import re
+import subprocess
+import sys
+
+_default_prefix = '${pcfiledir}/../..'
+
+_supported_compilers = ['gcc', 'clang']
+_supported_kernels = ['cygwin', 'linux', 'darwin']
+_supported_cpp_runtime_vendors = {'gnu': 'stdc++', 'llvm': 'c++'}
```

Comment:
IIRC the idea here was to identify which library a package used, in order to avoid conflicts. IOW, if my project is using libc++, and I am looking for package "Foo", don't use a "Foo" that is linked with libstdc++. Most of the platform stuff is along the lines of being able to tell if packages are ABI-compatible. AFAIK, pkg-config doesn't really have any equivalent for that, so there isn't much we can do but ignore it.

Actually, I guess that means we *could* drop the kernel/compiler stuff too, since right now it doesn't make a practical difference (unless I missed something), but I'll leave that up to you; I'm okay keeping it, also. The comment was mainly aimed at adding link flags for the C++ standard library.